packages/core/src/browser/shell/sidebar-bottom-menu-widget.tsx

```diff
@@ -1,88 +1,24 @@
-/********************************************************************************
- * Copyright (C) 2020 Alibaba Inc. and others.
- *
- * This program and the accompanying materials are made available under the
- * terms of the Eclipse Public License v. 2.0 which is available at
- * http://www.eclipse.org/legal/epl-2.0.
- *
- * This Source Code may also be made available under the following Secondary
- * Licenses when the conditions for such availability set forth in the Eclipse
- * Public License v. 2.0 are satisfied: GNU General Public License, version 2
- * with the GNU Classpath Exception which is available at
- * https://www.gnu.org/software/classpath/license.html.
- *
- * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
- ********************************************************************************/
-
-import { injectable, inject } from 'inversify';
-import * as React from 'react';
-import { ReactWidget } from '../widgets';
-import { ContextMenuRenderer } from '../context-menu-renderer';
-import { MenuPath } from '../../common/menu';
-
-export const SidebarBottomMenuWidgetFactory = Symbol('SidebarBottomMenuWidgetFactory');
-
-export interface SidebarBottomMenu {
-  id: string;
-  iconClass: string;
-  title: string;
-  menuPath: MenuPath;
-  order: number; // smaller one place lower
-}
-
+/* eslint-disable @typescript-eslint/tslint/config */
```

Comment:
I don't believe that this rule should be disabled here. Please restore the original header.